TestCases/parallel_regression.py

```diff
@@ -491,7 +491,7 @@ def main():
     actuatordisk_bem.cfg_dir = "rans/actuatordisk_bem"
     actuatordisk_bem.cfg_file = "actuatordisk_bem.cfg"
     actuatordisk_bem.test_iter = 15
-    actuatordisk_bem.test_vals = [-5.226958, -8.297197, -0.004325, -0.565388]
+    actuatordisk_bem.test_vals = [-5.282249, -10.335140, 0.001383, -0.375718]
```

Comment:
Does the case converge well? There is very little difference between your original values and the ones using the restart